abc.templates/drift-workflows/testdata/golden/basic/data/.github/workflows/guardian-drift-detection.yml

```diff
@@ -0,0 +1,112 @@
+# Copyright 2023 The Authors (see AUTHORS file)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Guardian Drift Detection is used to detect drift in GCP IAM and determine
+# if there are any unused/outdated terraform statefiles stored in GCS buckets.
+name: 'guardian_drift_detection'
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * 1' # Every Monday.
+
+permissions:
+  contents: 'read'
+  id-token: 'write'
+  issues: 'write' # For creating issues.
+
+env:
+  GITHUB_OWNER_NAME: '${{ github.event.repository.owner.login }}'
+  GITHUB_REPO_NAME: '${{ github.event.repository.name }}'
+  ORGANIZATION_ID: 'my-gcp-org-id'
+  GUARDIAN_VERSION: 'my-tag'
```

Comment:
Based on the inputs, I would expect this to be `1.0.0` if we update based on my other comment.

Reply:
It's coming up as `v1.0.0` rather than `1.0.0`, is this WAI?

Reply:
No, the v shouldnt be there...